### PR TITLE
added lunchmoney scanner

### PR DESCRIPTION
**Pull request by ladybug0125**

### CODE REVIEW

**Review by ahrav — APPROVED** [review]
lgtm

File: pkg/detectors/lunchmoney/lunchmoney.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"lunchmoney"}) + `\b([a-zA-Z0-9]{50})\b`)
```
Comment (ahrav) [inline pkg/detectors/lunchmoney/lunchmoney.go:24]: suggestion: tighten this regex and use `\b([a-f0-9]{50})\b`

**Review by ahrav — CHANGES_REQUESTED** [review]
Small tweak to the regex since lunchmoney only uses a-f

File: pkg/detectors/lunchmoney/lunchmoney.go (hunk 1 above)
Comment (ladybug0125) [reply pkg/detectors/lunchmoney/lunchmoney.go]: I changed the regex and pushed it. Thanks!

Comment (dustin-decker) [reply pkg/detectors/lunchmoney/lunchmoney.go]: Some extra context here Mildred, this is a hex-encoded credential which only uses a-f rather than a-z

**Pull request merged**